todocli/cli.py

```diff
@@ -111,14 +112,14 @@ def print_list(item_list, print_line_nums):
 
 
 def ls(args):
-    lists = todo_api.query_lists()
-    lists_names = [task_list["displayName"] for task_list in lists]
+    lists = [TodoList(x) for x in todo_api.query_lists()]
```

Comment:
I think it's better that `TodoList` objects are created in the `todo_api` instead of here. This file should be simply retrieving relevant data according to the input. So this line should go in `todo_api.query_lists()`.

Reply:
To clarify: I think this line should be `lists = todo_api.query_lists()` and all the TodoList object creation should go into the `query_lists()` function.

Reply:
Yes I agree, the only reason it isn't included here is because I've got another PR queued up where this is already done. I just included it in this PR as well.